package.json

```diff
@@ -31,6 +31,7 @@
     "nypr-auth": ">=0.2.0"
   },
   "devDependencies": {
+    "@types/ember": "^2.8.32",
```

Comment:
where did this come from?

Reply:
I intentionally added it, it's the Ember types definitions. It's meant for use w/ Typescript, but as a byproduct a lot of editors can use it for inline code completion because it documents Ember's various classes/arguments/methods.